shared/kubernetes/mizuTapperSyncer.go

```diff
@@ -310,6 +311,22 @@ func (tapperSyncer *MizuTapperSyncer) updateMizuTappers() error {
 
 	logger.Log.Debugf("Updating DaemonSet to run on nodes: %v", nodesToTap)
 
+	if nodes, err := tapperSyncer.kubernetesProvider.GetNodes(tapperSyncer.context, ""); err == nil {
+		for _, item := range nodes.Items {
+			if sliceutil.Contains(nodesToTap, item.Name) {
```

Comment:
is there a possibility that one of the 'nodesToTap' won't be in the 'nodes.Items', do we want to handle it?

Reply:
handled it better in https://github.com/up9inc/mizu/pull/1018/commits/92e2ea89dd5d807470d93b5123636b069ea9b1e1